tag.go

```diff
@@ -42,3 +48,129 @@ func (t Tag) TargetId() *Oid {
 func (t Tag) TargetType() ObjectType {
 	return ObjectType(C.git_tag_target_type(t.cast_ptr))
 }
+
+// CreateTagLightweight creates a new lightweight tag pointing to a commit.
+//
+// The name of the tag is validated for consistency (see git_tag_create() for the rules
+// https://libgit2.github.com/libgit2/#HEAD/group/tag/git_tag_create) and should
+// not conflict with an already existing tag name.
+//
+// If force is true and a reference already exists with the given name, it'll be replaced.
+//
+// The created tag is a simple reference and can be queried using
+// repo.References.Lookup("refs/tags/<name>"). The name of the tag (eg "v1.0.0")
+// is queried with ref.Shorthand().
+func (repo *Repository) CreateTagLightweight(name string, commit *Commit, force bool) (*Oid, error) {
```

Comment:
This needs to explain what the return Oid is, there is no way to know what it is from the signature.